app/_layouts/default.html

```diff
@@ -35,8 +35,7 @@
       <ul class="wrap list-linear-pipes">
         <li>
           <a href="http://github.com/centresource/generator-playbook">GitHub</a>
-        </li>
-        <li>
+        </li><li>
```

Comment:
The same issue we've dealt with in the past where whitespace is displayed as spacing on the screen. Open to more elegant solutions.